features/datum.feature

```diff
@@ -228,51 +231,49 @@ Functionaliteit: leveren van een datum
       | JaarMaandDatum | 20200300 |            | 2020 | 3     |          | maart 2020   |
 
     Abstract Scenario: <type> in inOnderzoek datumIngangOnderzoek
-      Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                                        | waarde     |
-      | burgerservicenummer                         | 555550001  |
-      | aanduiding gegevens in onderzoek (01.83.10) | 010000     |
-      | datum ingang onderzoek (01.83.20)           | <GbaDatum> |
-      Als personen wordt gezocht met de volgende parameters
-      | naam                | waarde                          |
-      | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 555550001                       |
-      | fields              | <groep>.<veld>                  |
-      Dan heeft de response een persoon met de volgende '<groep>' gegevens
-      | naam               | waarde        |
-      | <veld>.type        | <type>        |
-      | <veld>.datum       | <datum>       |
-      | <veld>.jaar        | <jaar>        |
-      | <veld>.maand       | <maand>       |
-      | <veld>.onbekend    | <onbekend>    |
-      | <veld>.langFormaat | <langFormaat> |
+      Gegeven de persoon met burgerservicenummer '000000188' heeft de volgende gegevens
+        | naam                              | waarde     |
+        | aanduiding in onderzoek (83.10)   | 010000     |
+        | datum ingang onderzoek (83.20)    | <GbaDatum> |
+        Als personen wordt gezocht met de volgende parameters
+        | naam                | waarde                          |
+        | type                | RaadpleegMetBurgerservicenummer |
+        | burgerservicenummer | 000000188                       |
+        | fields              | <groep>.<veld>                  |
+        Dan heeft de response een persoon met de volgende '<groep>' gegevens
+        | naam               | waarde        |
+        | <veld>.type        | <type>        |
+        | <veld>.datum       | <datum>       |
+        | <veld>.jaar        | <jaar>        |
+        | <veld>.maand       | <maand>       |
+        | <veld>.onbekend    | <onbekend>    |
+        | <veld>.langFormaat | <langFormaat> |
 
       Voorbeelden:
-      | groep       | veld                 | type           | GbaDatum | datum      | jaar | maand | onbekend | langFormaat  |
+      | groep       | veld                        | type           | GbaDatum | datum      | jaar | maand | onbekend | langFormaat  |
```

Comment:
De 1e 2 kolommen van deze tabel kunnen verwijderd worden. De waardes kunnen hard in het scenario geplaatst worden aangezien ze voor alle voorbeelden gelijk zijn.

Reply:
Aangepast